stream_chat/channel.py

```diff
@@ -135,6 +135,10 @@ def mark_read(self, user_id: str, **data: Any) -> StreamResponse:
         payload = add_user_id(data, user_id)
         return self.client.post(f"{self.url}/read", data=payload)
 
+    def mark_unread(self, user_id: str, **data: Any) -> StreamResponse:
+        payload = add_user_id(data, user_id)
+        return self.client.post(f"{self.url}/read", data=payload)
```

Comment:
this url seems incorrect (read vs unread)
are you sure the base of the url is correct too?

Reply:
Thanks :) copy&paste error from the above method, fixed